engine/src/main/java/org/terasology/physics/engine/PhysicsSystem.java

```diff
@@ -242,13 +244,55 @@ public void update(float delta) {
 
         for (CollisionPair pair : collisionPairs) {
             if (pair.b.exists()) {
-                pair.a.send(new CollideEvent(pair.b,pair.pointA,pair.pointB,pair.distance,pair.normal));
+                short bCollisionGroup = getCollisionGroupFlag(pair.b);
+                short aCollidesWith = getCollidesWithGroupFlag(pair.a);
+                if((bCollisionGroup & aCollidesWith) != 0 || 
+                        (pair.b.hasComponent(BlockComponent.class) && !pair.a.hasComponent(BlockComponent.class))){
```

Comment:
This check would mean that no two items colliding would ever mention the collision took place. Is that intended?